records_mover/records/records_format.py

```diff
@@ -60,20 +63,21 @@ def __eq__(self, other: object) -> bool:
                     self.hints == other.hints)
         return False
 
-    def alter_hints(self, new_hints: RecordsHints) ->\
+    def alter_hints(self, new_hints: UntypedRecordsHints) ->\
             'DelimitedRecordsFormat':
         input_hints = dict(self.hints)  # make copy
         input_hints.update(new_hints)
         return DelimitedRecordsFormat(variant=self.variant,
-                                      hints=input_hints)
+                                      hints=input_hints)  # type: ignore
 
     def alter_variant(self, variant: str) -> 'DelimitedRecordsFormat':
         return DelimitedRecordsFormat(variant=variant,
-                                      hints=self.hints)
+                                      hints=self.hints)  # type: ignore
 
+    # TODO: Can this be made to return PartialRecordsHints?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker